src/main/java/com/example/codebase/domain/notification/repository/NotificationSettingRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.example.codebase.domain.notification.repository;
+
+import com.example.codebase.domain.member.entity.Member;
+import com.example.codebase.domain.notification.entity.NotificationSetting;
+import com.example.codebase.domain.notification.entity.NotificationSettingIds;
+import com.example.codebase.domain.notification.entity.NotificationType;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface NotificationSettingRepository extends JpaRepository<NotificationSetting, NotificationSettingIds> {
+
+    @Query("SELECT n FROM NotificationSetting n LEFT JOIN Member m ON n.member = m WHERE n.member = :member")
```

Comment:
NotificationSetting 내부에 member_id 가 있으므로 해당 LEFT JOIN은 불필요하네요!

```suggestion
    @Query("SELECT n FROM NotificationSetting n WHERE n.member = :member")
```